tests/300_mongodb_import_connect.robot

```diff
@@ -8,44 +8,44 @@ Suite Setup         Set Library Search Order    SeleniumLibrary
 Suite Teardown      Tear Down The Test Suite
 Test Setup          Given Setup The Test Case
 Test Teardown       Tear Down The Test Case
-Force Tags          UI      mongo
+
+Force Tags          UI    mongo
 
 
 *** Test Cases ***
 Scenario: Import MongoDB Provider Account From Developer View
-    [Tags]    smoke    RHOD-10
-    When User Navigates To Add Topology Screen From Developer View      MongoDB
+    [Tags]    RHODA-006    smoke
+    When User Navigates To Add Topology Screen From Developer View    MongoDB
     And User Navigates To Import Provider Account Screen From Developer View
     And User Enters Data To Import MongoDB Provider Account
     Then Provider Account Import Success
 
 Scenario: Verify error message for invalid credentials on MongoDB
-    [Tags]    smoke    RHOD-49-1
+    [Tags]    RHODA-007    smoke
     When User Filters Project ${operatorNamespace} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     And User Enters Invalid Data To Import MongoDB Database Provider Account
     Then Provider Account Import Failure
 
 Scenario: Import MongoDB Provider Account From Administrator View
-    [Tags]    smoke    RHOD-46
+    [Tags]    RHODA-008
```

Comment:
Shouldn't this also be a smoke test?

Reply:
@ikanias This flow would be covered on next test script execution on the same test suite. So I skipped this one.